base-dockerfile

```diff
@@ -0,0 +1,29 @@
+# Source: https://github.com/LukeMathWalker/cargo-chef
+
+FROM lukemathwalker/cargo-chef:latest-rust-1 AS chef
+WORKDIR /misarch-review
+
+FROM chef AS planner
+COPY . .
+RUN cargo chef prepare --recipe-path recipe.json
+
+FROM chef AS builder 
+COPY --from=planner /misarch-review/recipe.json recipe.json
+
+RUN apt update && apt install -y protobuf-compiler && rm -rf /var/lib/apt/lists/*
+
+# Build dependencies - this is the caching Docker layer!
+RUN cargo chef cook --release --recipe-path recipe.json
+# Build application
+COPY . .
+
+RUN cargo build --release --bin misarch-review
+
+# We do not need the Rust toolchain to run the binary!
+FROM debian:bookworm-slim AS runtime
+
+RUN apt update && apt install -y protobuf-compiler && rm -rf /var/lib/apt/lists/*
```

Comment:
just out of curiosity
with the dapr library gone
do we even need this anymore?

Reply:
well I should have written this in the other PR...

Reply:
No, not necessary anymore, will remove this.